networkit/test/test_clique.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+import unittest
+import os
+import networkit as nk
+import numpy as np
+import scipy
+
+class Test_Cliques(unittest.TestCase):
+
+	def test_maxClique(self):
+		G = nk.readGraph("input/jazz2_directed.gml",nk.Format.GML)
+		G.removeSelfLoops()
+		A = nk.clique.MaximalCliques(G, callback=[1,2]).run().getCliques()
```

Comment:
According to the [documentation](https://networkit.github.io/dev-docs/python_api/clique.html), `callback` should be a callable, but here you are passing a list. Please clarify what are you trying to test here.